integrations/extensions/starter-kits/language-model-complex-informational-tasks/README.md

```diff
@@ -0,0 +1,71 @@
+- `last_action_accessed`: Used by "No Action Matches" to support multiple conversational turns on the same topic. For example the "Internet Plan Questions" action sets this value to "Internet Plan Questions" so after the first internet plan query, all subsequent queries will be fall back to the Internet Plan Questions action if they do not directly match some other action. This is meant to illustrate extended conversational interaction. It is a current limitation of the starter kit that once this value is set, it does not automatically get reset when the conversation moves on past this topic.
+- `messages` : Input to the OpenAI model.
+- `model_for_chat` : The OpenAI model used, defaults to `gpt-3.5-turbo`.
+- `model_max_tokens` : The maximum number of text fragments to input to the model. The starter kit uses 200.
+- `model_response`: The text generated by the model in response to the model input `messages`.
+- `tv_plan_messages`: Holds information from the TV Plan Questions action that is used as input to the model.
+- `tv_plan_prompt`: The prompt for the OpenAI model on the topic of TV plans. It includes instructions and data for the model to use when generating an answer.
+- `user_message`: This is used to add the user query to the model messages `{ "role": "user",  "content": input.text}`.
+- `verbose`: When set to `True` prints debugging information. Default is `False`. In this starter kit, when `verbose` is `True`, a step in the "Generate Answer" action will print out the prompt that is being sent to the model.
+
+### Example usage
+
+#### An example of how to use the `Purchase a Plan` action for this starter kit:
+
+<img src="./assets/complex-info-openai-upgrade-plan-example.png" width="300"/>
+
+#### An example of how to use the `TV Plan Questions` action for this starter kit:
+
+<img src="./assets/complex-info-openai-tv-plan-example.png" width="300"/>
+
+## Use at your own risk
+
+You can get the actions to just respond to _anything_ a user says by updating "No action matches" (and optionally "Fallback") built-in actions in Watson Assistant to set `model_prompt` to `input.text`, call one of these actions, and then put the `model_response` in "Assistant says". With more work, you can even keep track of things the assistant and the user have said to each other and put them into the prompt too, ideally as separate entries in the `messages` list of the chat completion API as described in the [chat completion API documentation](https://platform.openai.com/docs/api-reference/chat); the result would be something similar to the ChatGPT experience in which you would have an ongoing conversation with the model. This can be a fun thing to try out, but _we strongly recommend that you do not do this for any serious chatbot that you use to represent your enterprise_, since you would have no control over what it will say.
```

Comment:
Proposed rewrite:

The prompts in this kit provide specific details for the models to pull information from, and often the models are quite good at focusing on those details.  However, depending on the question, the details of the prompt, and the model used, the models can also provide additional information that is not in the prompt and may not always be what you want your assistant to say.  We strongly recommend that anyone using this approach do a great deal of testing, and think carefully before deploying something like this.  It can allow you to get some impressive results very quickly and easily, but it comes with some significant risk of producing misinformation in some cases.  Careful revision of the prompts can reduce that risk, but probably will not eliminate it completely.

Reply:
Sounds good. I have updated accordingly, thanks.